async_plus/retry.py

```diff
@@ -0,0 +1,43 @@
+import asyncio
+import itertools
+import time
+
+
+__all__ = ['RetryDelayer']
+
+
+class RetryDelayer:
+    """
+    Usage example:
+
+        retry_delayer = async_plus.RetryDelayer()
+        while True:
+            try:
+                await run_service_x()
+            # In Python <3.8 it inherits from Exception
+            except asyncio.CancelledError:
+                raise
+            except Exception:
+                logger.exception('Error in service X:')
+                await retry_delayer.sleep()
+    """
+
+    def __init__(self, delays=(0, 10, 60), reset_after=None):
```

Comment:
why do you prefer to use a sequence of delays instead of [exponential backoff](https://en.wikipedia.org/wiki/Exponential_backoff)?